x-pack/legacy/plugins/reporting/types.d.ts

```diff
@@ -134,6 +124,19 @@ export interface JobDocPayload {
   objects?: string | null; // string if completed job; null if incomplete job;
 }
 
+export type JobDocPayloadScreenshot = JobDocPayload & {
+  browserTimezone: string;
+  layout: any;
+};
```

Comment:
Move this to:
`x-pack/legacy/plugins/reporting/export_types/png/types.d.ts` OR
`x-pack/legacy/plugins/reporting/export_types/printable_pdf/types.d.ts`

Reply:
or have it duplicated in both maybe